metabase/metabase
# Add basic layout for the new home page

Epic: https://github.com/metabase/metabase/issues/21322

How to test:
- Run an OSS version. Open homepage. Make sure the bridge image is rendered correctly.
- Run an EE version. Enable whitelabeling and make sure the bridge image is not rendered.

<img width="968" alt="Screenshot 2022-03-31 at 16 16 58" src="https://user-images.githubusercontent.com/8542534/161064014-3c961149-bf53-4f94-8072-6eb10d4a1210.png">
<img width="875" alt="Screenshot 2022-03-31 at 16 17 32" src="https://user-images.githubusercontent.com/8542534/161064027-2380d29b-bfcd-4019-bb00-3a1ed008a4a2.png">
